### DISCUSSION TITLE
Welcome to assemblylift Discussions!

### DISCUSSION THREAD

**Discussion by dotxlem** [akkoro/assemblylift#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OAyomide** [comment]
Hi! 🙃

My name is Ayomide. I am  software engineer, currently at Enyata – a software engineering consulting and development company; located in Lagos, Nigeria. I am a "baby" rustacean but I have working understanding of rust even though I still fight the borrow checker 🤣 .

I am interested in contributing to Assemblylift and hopefully help bring it to mainstream. I love WebAssembly and Rust so its a no brainer why this community is the perfect fit for me.

Thanks!!
